guides/v3.7.0/tutorial/autocomplete-component.md

```diff
@@ -512,11 +507,10 @@ test('should update with matching listings', async function (assert) {
   await fillIn(this.element.querySelector('.list-filter input'),'s');
   // keyup event to invoke an action that will cause the list to be filtered
   await triggerKeyEvent(this.element.querySelector('.list-filter input'), "keyup", 83);
+  await settled();
 
-  return settled().then(() => {
-    assert.equal(this.element.querySelectorAll('.city').length, 1, 'One result returned');
-    assert.equal(this.element.querySelector('.city').textContent.trim(), 'San Francisco');
-  });
+  assert.equal(this.element.querySelectorAll('.city').length, 1, 'One result returned');
+  assert.equal(this.element.querySelector('.city').textContent.trim(), 'San Francisco');  
```

Comment:
the code change here is correct, though I'm wondering why we even use `settled()` here. `triggerKeyEvent()` already calls `settled()` before returning, so this seems redundant.

Reply:
I believe this example is the introduction of the `settled` helper. Since it’s redundant, maybe it would make since to remove it and move the introduction of `settled` to a test where it’s needed?

Reply:
@locks ☝️ ?

Reply:
@locks @Turbo87 If you think it makes sense, I can remove the `settled` helper that is used after `tiggerKeyEvent()`

Reply:
Turns out I commented on the same issue over in https://github.com/ember-learn/super-rentals/pull/117. IMHO, we should remove these superfluous `await settled()` calls (any `await settled()` after another async test helper from `@ember/test-helpers` are generally not needed).

Reply:
I'm fine removing `await settled()`, but as I mentioned above, I don't believe there is another time in the tutorial where that test helper is introduced.

Previously, were waiting on a response from @locks about his opinion on removing `await settled()`, so in the meantime, I updated the super-rentals application to be consistent with this PR as @jenweber suggested.